src/plugins/drawing/model/drawing-content.ts

```diff
@@ -60,14 +61,18 @@ export const DrawingContentModel = NavigatableTileModel
     get objectMap() {
       // TODO this will rebuild the map when any of the objects change
```

Comment:
Is this old TODO comment still relevent?

Reply:
Yeah, the idea was to be able to update the map as things change rather than always rebuild it from scratch. I didn't change anything there.